ignite/pkg/chaincmd/runner/chain.go

```diff
@@ -57,6 +57,18 @@ func (r Runner) InPlace(ctx context.Context, newChainID, newOperatorAddress stri
 	)
 }
 
+func (r Runner) MultiNode(ctx context.Context, options ...chaincmd.MultiNodeOption) error {
+	runOptions := runOptions{
+		stdout: os.Stdout,
+		stderr: os.Stderr,
+	}
+	return r.run(
+		ctx,
+		runOptions,
+		r.chainCmd.TestnetMultiNodeCommand(options...),
```

Comment:
Can we run all the node with the following log level: `"*:error,p2p:info,state:info"`